storage/container_storage_interface/persistent-storage-csi-ember.adoc

```diff
@@ -0,0 +1,29 @@
+[id="persistent-storage-csi-ember"]
+= Ember-CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ember
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Ember-CSI.
```

Comment:
```suggestion
{product-title} is capable of provisioning persistent volumes (PVs) by using the Container Storage Interface (CSI) driver for Ember-CSI.
```

Small suggestion here to add 'by' because 'using' on its own can have two different translations depending on the meaning. Even though it reads a little bit more clunkily, changing 'using' for either 'by using' or 'that uses' (depending) improves ease of translation/localization especially for machine translation as well improves clarity for non-native English readership (the majority of RH readers). 
See IBM Style Guide for more info.